apps/portal-api/src/modules/services/instances/domain.ts

```diff
@@ -12,15 +15,16 @@ export const serviceInstanceDomain = {
   createPlatformServiceInstance: async (
     context: PortalContext,
     serviceDefinitionId: string,
-    platformIdentifier: PlatformIdentifier
+    platformIdentifier: PlatformIdentifier,
+    status: string = ServiceInstanceCreationStatus.Ready
   ): Promise<ServiceInstanceId> => {
     const id = uuidv4() as ServiceInstanceId;
     await db(context, 'ServiceInstance').insert([
       {
         id,
         name: serviceInstanceNameMappedByPlatformIdentifier[platformIdentifier],
         description: '',
-        creation_status: 'READY',
+        creation_status: status,
```

Comment:
If you apply the suggestion with the function parameters:
```suggestion
        creation_status,
```

Reply:
done in [f19ea09](https://github.com/FiligranHQ/xtm-hub/pull/1102/commits/f19ea09d05939af65f1ad757710dbc51dca16a14)